src/main/java/configuration/ThreadCountConfig.java

```diff
@@ -0,0 +1,24 @@
+package configuration;
+
+import java.util.regex.Pattern;
+
+/***
+ * Set up the number of test run threads
+ */
+public class ThreadCountConfig {
+
+    private static final Pattern isInteger = Pattern.compile("^\\d+$");
+
+    public static int getThreads() {
+        String threadCount = System.getProperty("threadNumber");
+        return getThreadNumber(threadCount);
+    }
+
+    private static int getThreadNumber(String strNum) {
```

Comment:
strNum is not a clear argument name, i think it's worth updating

Reply:
Also i think it makes sense to update method's name to smth like _getThreadNumberFrom_

Reply:
fixed